### PR TITLE
apps/st_things : rename GPIO config from iotbus to st_things

### PR DESCRIPTION
**Pull request by YiHyunJin**
…nd EXAMPLES_IOTBUS_TEST.

Both features depends on EXAMPLES_IOTBUS_TEST_RED_LED_GPIO_NUM.
To solve this, EXAMPLES_IOTBUS_TEST_RED_LED_GPIO_NUM is changed to EXAMPLES_ST_THINGS_RED_LED_GPIO_NUM.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [a9697d0f72fde72c7babc2cb975559a441a2723f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a9697d0f72fde72c7babc2cb975559a441a2723f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0e7fccf452aa3966b7601bfc6dbd931129e477b8] -

**Comment by seinfra** [comment]
Target : [54ff809c148cef313810a8c0c34379ed80446bed] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [54ff809c148cef313810a8c0c34379ed80446bed] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b916dbab1357993fce749013d6a5fec6265ee590] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [b916dbab1357993fce749013d6a5fec6265ee590] - Code Rule Check (C++) OK.

File: apps/examples/st_things/blink/blink.h [hunk 1]
```diff
@@ -21,11 +21,11 @@
 #include <tinyara/gpio.h>
 #include <iotbus/iotbus_gpio.h>
 
-#if !defined(CONFIG_EXAMPLES_IOTBUS_TEST_RED_LED_GPIO_NUM)
+#if !defined(EXAMPLES_ST_THINGS_RED_LED_GPIO_NUM)
```
Comment (sunghan-chang) [inline apps/examples/st_things/blink/blink.h]: You missed ```CONFIG_```.

**Comment by seinfra** [comment]
Target : [b4ed33cd3f3d78054d204897e1853efd0cb87888] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [b4ed33cd3f3d78054d204897e1853efd0cb87888] - Code Rule Check (C++) OK.

**Pull request merged**